src/harness/certs/README.md

```diff
@@ -103,6 +104,9 @@ Required certificates are:
 * `wrong_type_client.cert`: leaf CBSD certificate signed using server.csr 
   Used on security test test_WINNF_FT_S_SCS_10.
   
+* `blacklisted_client.[cert|key]`: leaf CBSD device certificate is blacklisted 
```

Comment:
This isn't quite true, since the name changes depending on whether it's a CBSD, DP, or SAS cert.

Reply:
Updated   README.md for CBSD,DP and SAS as well.